test/index.test.ts

```diff
@@ -139,6 +140,60 @@ describe('index', () => {
 
 	});
 
+	describe('closeChannel', () => {
+
+		it('should close the channel if connect with publish channel has been called', async () => {
+
+			// Given
+			transport = new Transport({
+				url: 'amqp://localhost'
+			});
+
+			sinon.stub(Publisher.prototype, 'init');
+			sinon.stub(Publisher.prototype, 'publish');
+
+			await transport.connect({ url: 'testUrl' });
+
+			await transport.connect();
+
+			// When
+			await transport.closeChannel();
+			await transport.close();
```

Comment:
I don't think the unit test for `closeChannel` should call `close`.

Reply:
Due latest modifications all the functionality to close the channels is included in close function